src/Renci.SshNet/NetConfClient.cs

```diff
@@ -21,8 +21,8 @@ public class NetConfClient : BaseClient
         /// Gets or sets the operation timeout.
         /// </summary>
         /// <value>
-        /// The timeout to wait until an operation completes. The default value is negative
-        /// one (-1) milliseconds, which indicates an infinite time-out period.
+        /// The timeout to wait until an operation completes. The default value is <see cref="SshNet.Session.InfiniteTimeSpan"/>

```

Comment:
Don't you think it would be better to expose those constants since they are static readonly structs anyway, so can't be changed, so that consumers of the class don't need to define their own infinite constants?

I've got the reverted code ready to go, just haven't added it to the request yet, until I hear what you think.